source/features/first-published-tag-for-merged-pr.tsx

```diff
@@ -11,7 +11,7 @@ import createBanner from '../github-helpers/banner';
 import TimelineItem from '../github-helpers/timeline-item';
 import attachElement from '../helpers/attach-element';
 import {canEditEveryComment} from './quick-comment-edit';
-import {buildRepoURL, getRepo} from '../github-helpers';
+import {buildRepoURL, getRepo, isRefinedGitHubRepo} from '../github-helpers';
 import onConversationHeaderUpdate from '../github-events/on-conversation-header-update';
```

Comment:
```suggestion
import onConversationHeaderUpdate from '../github-events/on-conversation-header-update';
import {buildRepoURL, getRepo, isRefinedGitHubRepo} from '../github-helpers';
```

Reply:
I think we should stop sorting imports manually 😅 ❌ 🌲